contrib/objectstore/subscriber/marshaller/marshaller.go

```diff
@@ -0,0 +1,25 @@
+package marshaller
+
+import (
+	"fmt"
+	"github.com/skydive-project/skydive/contrib/objectstore/subscriber/marshaller/custom1"
+	"github.com/skydive-project/skydive/flow"
+)
+
+// Marshaller allows generic transformations of a flow
+type Marshaller interface {
+	// Marshal transforms a flow before being stored
+	Marshal(flow *flow.Flow) interface{}
+}
+
+// New creates a new flow storage based on the backend
+func New(marshallerName string) (Marshaller, error) {
+	switch marshallerName {
+	case "custom1":
+		return custom1.New(), nil
+	case "":
```

Comment:
do we need this case ?

Reply:
We want the default to be no transformation at all.
The default is an empty string, so we use it to indicate no transformation.